core/templates/dev/head/player/exploration_player.html

```diff
@@ -40,15 +40,50 @@
           <span class="glyphicon glyphicon-comment"></span>
         </a>
       </li>
-      {% if can_edit %}
-        <li>
+      <li>
+        {% if can_edit %}
           <a ng-href="/create/<[explorationId]>" tooltip="Edit" tooltip-placement="bottom" target="_blank">
             <span class="glyphicon glyphicon-pencil"></span>
           </a>
-        </li>
-      {% endif %}
+        {% elif username %}
+          <a ng-click="showLearnerSuggestionModal()" tooltip="Suggest Edit" tooltip-placement="bottom" target="_blank">
```

Comment:
I suggest "Suggest a Change" as the tooltip. The current tooltip text reads as two separate actions (especially since the content ends up being on two separate lines.)